### ISSUE TITLE
/tmp/DATS_START_UP_IN_PROGRESS Crash Backend

### ISSUE THREAD

**Issue by AhmadHAW** [uhh-lt/dats#469]
This bug can happen out of nowhere, because other developers on dwts may have crashed their loading process before the temp file was deleted. Need sudo rights to delete the zombie file.

2024-11-19 09:54:52.439 | INFO     | app.core.startup:startup:54 - Booting Discourse Analysis Tool Suite Backend ...
2024-11-19 09:54:54.212 | DEBUG    | app.core.startup:is_startup_in_progress:36 - Startup indicator file already exists at: /tmp/DATS_START_UP_IN_PROGRESS!
2024-11-19 09:54:54.212 | DEBUG    | app.core.startup:startup:58 - Waiting for API startup process to be completed...

Robert meint du hast dich schon an dem Bug versucht?

**Comment by floschne** [comment]
Yes, that issue might occur if two or more devs start the DATS at the same moment. One solution would be to generate a unique id per DATS instance, which is added to the startup file name.

**Comment by AhmadHAW** [comment]
This solves one of the two General Problems. The first Problem Happens, when two Services are started at the Same time. The second Problem occurs, when the cleanup is aborted. Does this happens If one mashes the Stop Button/kill command multiple Times or can IT also Happen If the kill command is send once but at the wrong time? In other words, does the Server have a gracious shutdown during Startup?

**Comment by AhmadHAW** [comment]
Accidently closed

**Comment by AhmadHAW** [comment]
Idk why ITS closing all the time

**Comment by bigabig** [comment]
just put that random id in the backend/.env file. could be an easy fix, no?

**Comment by AhmadHAW** [comment]
Yes, this should do the Trick. I guess the Zombie files that get created can be deleted manually from time to time.
